ansible/roles/commcarehq/templates/localsettings.py.j2

```diff
@@ -255,15 +277,6 @@ SOIL_DEFAULT_CACHE = "redis"
 
 REPORT_CACHE = 'redis'
 
-{% if shared_drive_enabled|default(true) %}
-SHARED_DRIVE_ROOT = '{{ shared_mount_dir }}'
-{% else %}
-SHARED_DRIVE_ROOT = '{{ shared_data_dir }}'
-{% endif %}
-RESTORE_PAYLOAD_DIR_NAME = '{{ restore_payload_dir_name }}'
-TRANSFER_FILE_DIR_NAME = '{{ transfer_payload_dir_name }}'
-SHARED_TEMP_DIR_NAME = '{{ shared_temp_dir_name }}'
-TRANSFER_SERVER = '{{ (proxy_type == 'nginx') | ternary('nginx', 'apache') }}'
```

Comment:
maybe @dannyroberts moved it when he was changing localsettings. I dont think i moved it. not sure it changes things though